index.js

```diff
@@ -0,0 +1,94 @@
+const BASE_URL = 'https://trektravel.herokuapp.com';
+$(document).ready(() => {
+
+  $('#load').click(function(response) {
+
+    let fullUrl = BASE_URL + '/trips'
+    $.get(fullUrl, response => {
+      console.log(response);
+
+      response.forEach((trip) => {
+        let name = trip.name,
+        id = trip.id,
+        continent = trip.continent,
+        weeks = trip.weeks;
+        let allTripInfo = `<li data-id=${id}><h3>${name}:</h3> <p> Location: ${continent} Length: ${weeks} weeks. </p></li>`;
+        $('#all-trips ul').append(allTripInfo);
+      });
+
+      $('#all-trips h3').on('click', onTripClick)
+    })
+    .fail(function(response){
+      console.log(response);
+      console.log('failure');
+      $('#message').html(`<p>Error!</p>`)
+    })
+    .always(function(){
+      console.log('always even if we have success or failure');
+    });
+  });
+
+  // begin .get to see more info on individual trips
+  const onTripClick = function onTripClick() {
+    const tripLi = $(this.parentElement)
+    let tripID = tripLi.attr('data-id');
+    let individualURL = `${BASE_URL}/trips/${tripID}`
+
+    $(this).click((event) => {
+      event.stopPropagation();
+    }); // stopping click from running too many times
+
+    $.get(individualURL, response => {
+
+      tripLi.append(`
+        <p>ID: ${response.id}</p>
+        <p>Category: ${response.category}</p>
```

Comment:
It would probably make sense to wrap this generated HTML in a sectioning element. Even dynamic content needs to be semantic!